credmark/cmf/model/context.py

```diff
@@ -46,7 +46,10 @@ def __init__(self, context, prefix: str, block_number: Union[int, None] = None):
                 self.__doc__ = f'Run a model.\n\nAvailable models: {", ".join(slugs)}'
 
     # run a model. args can be a positional DTO or dict or kwargs
-    def __call__(self, input: Union[DTO, dict, None] = None, return_type=None, **kwargs) -> dict:
+    def __call__(self,
+                 input: Union[DTO, dict, None] = None,
+                 return_type: Union[DTO, dict, None] = None,
+                 **kwargs):
```

Comment:
@leafyoung Is it getting the return type right without the hint?

Reply:
@iwyrkore Earlier I thought of relying on `self.__context.run_model` without a type hint. Now I have added the type hints.